tests/ja4-quic/test.yaml

```diff
@@ -0,0 +1,16 @@
+requires:
+  min-version: 8.0.0
+  files:
+    - rust/src/ja4.rs
```

Comment:
Do not need to check the file, if you require the feature, right ?
Thinking about if the file gets moved to rust/src/crypto/ja4.rs for instance

Reply:
True, changed.